### DISCUSSION TITLE
30/5/24

### DISCUSSION THREAD

**Discussion by Pamela198713** [enbalde-ispc/enbalde-ispc#199] [Daily Sprint]
Que hice
Que hago
Impedimento

**Comment by Pamela198713** [comment]
Ayer estudié para rendir emprendedurismo
Hoy me voy a organizar para hacer pantallas sigo engripada en cama
Resfrío,el desgano, malestar

**Comment by rpgrca** [comment]
Ayer hice el examen de emprendedurismo y estuve en la clase de tecnología y desarrollo. Hoy voy a estar en la clase de práctica profesionalizante. Espero volver con la parte de Android para hacer la sección de contacto y el carrito de compras pero estoy tapado de trabajo en el laburo así que intentaré.

**Comment by LuciaL152** [comment]
Ayer estuve estudiando para hacer el examen de emprendedurismo, hoy estuve trabajando, ahora estoy en la clase  de pp y después me voy a poner con android.

**Comment by fmdcba** [comment]
Ayer con el examen y hoy pude asistir a clases, pero no metí mano en el proyecto. Mañana voy a estar más desocupado, así que me ocuparé en las entregas pendientes.
